src/libais/ais.h

```diff
@@ -1566,9 +1566,49 @@ class Ais8_200_10 : public Ais8 {
   Ais8_200_10(const char *nmea_payload, const size_t pad);
 };
 
-// 21 and 22 do not exist
+// vessel_traking_and_tracing_standard_ed1-2_ccnr_23-apr_2013_en.pdf
+// ETA at lock/bridge/terminal
+// Todo : complete this class
+class Ais8_200_21 : public Ais8 {
+ public:
+  string country;         // UN country code                    12 bits
+  string location;        // UN location code                   18 bits
+  string section;         // Fairway section number             30 bits
+  string terminal;        // Terminal code                      30 bits
+  string hectometre;      // Fairway hectometre                 30 bits
```

Comment:
What does this mean?  Is it really supposed to 100's of meters or ?  And if it is a length, of what?

Found nothing much of use in https://www.ccr-zkr.org/files/documents/ris/vtt12_e.pdf

Reply:
This is the distance in hectometre (100's of meters) from the beginning of the fairway.

> Fairway section hectometre (5 digits)
> The hectometre identifies a specific bridge, lock or berth.

ATLNZ00001GAUGE21352 Austria, Linz, Danube, gauge at river-km 2135,2

Reply:
Does the properties are correctly named?

Standards | field
--|--
UN country code | country
UN location code | code
Fairway section number | section
Terminal code | terminal
Fairway hectometre | hectometre
… | …